src/pages/UserPage/UserPage.tsx

```diff
@@ -29,6 +35,23 @@ const UserPage: FC = () => {
 
   const userNameString = `${user.user.forename} ${user.user.surname}`
 
+  const handleArchiveModal = () => {
+    const isMainUser = some(user?.roles, (mainUser) => mainUser?.is_root)
+
+    !isMainUser &&
+      showModal(ModalTypes.UserAndRoleManagement, {
+        title: t('modal.archive_user_account'),
+        cancelButtonContent: t('button.no'),
+        proceedButtonContent: t('button.yes'),
```

Comment:
Currently I have seen that in user and role management modals we have yes and no buttons. Do we have anything else as well? If there is mostly yes and no then these should be default values.

Reply:
Yes, there is mostly "yes" and "no", there is also "cancel" and "activate". Will add the default values.